src/main/java/userinterface/UserInterface.java

```diff
@@ -144,25 +146,25 @@ public static void main(String[] args) {
     } // main
 
 
-    /**
-     * This enum class contains values for number of RF Ports
-     * required for LTE and WCDMA.
-     */
-    enum RF_PORT_NUMBER {
-        LTE_RF_PORTS_NUMBER(4),
-        WCDMA_RF_PORTS_NUMBER(2);
-
-        private final int value;
-
-        RF_PORT_NUMBER(int value) {
-            this.value = value;
-        }
-
-        public int getValue() {
-            return value;
-        }
-
-    }
+//    /**

```

Comment:
To check, are you keeping these as commented out since you'd like to keep them as reference, or should these be removed?